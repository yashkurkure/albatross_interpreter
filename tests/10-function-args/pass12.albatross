fun f2 void () {
  return;
}

f2();
return 473;
