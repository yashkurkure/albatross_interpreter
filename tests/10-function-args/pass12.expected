Function declared "f2" returns void
Function called "f2" returns void
